examples/guestbook/README.md

```diff
@@ -83,15 +158,17 @@ kind: ReplicationController
 metadata:
   name: redis-master
   labels:
-    name: redis-master
+    app: redis
+    tier: backend
+    role: master
 spec:
   replicas: 1
```

Comment:
We could put YAML comments into the config. 

1 replica is the default, but I agree that it makes sense to be explicit in this case.